tools/docker/Dockerfile

```diff
@@ -1,6 +1,23 @@
 FROM alpine
```

Comment:
Consider distroless:static-nonroot if this is just a go binary

Reply:
it actually uses one Unix command to set TTY parameters, I thought about it, perhaps if we can remove this dependency, we can move to distroless.

Reply:
fixed that.